src/data/faq_de.json

```diff
@@ -1804,7 +1804,8 @@
                         "anchor": "recovered",
                         "active": false,
                         "textblock": [
-                            "Seit der Version 1.13 haben Sie die Möglichkeit nach einer überstanden SARS-CoV-2 Infektion, die App wieder zu aktivieren und die Fortführung der Risiko-Ermittlung nach Teilung der eigenen Zufalls-IDs erneut zu starten. Dies bedeutet, dass Sie nun entscheiden können, ob und wann Sie die Risiko-Ermittlung wieder einschalten, nachdem Sie Ihre verschlüsselten Zufalls-IDs nach einem Positivtest geteilt und andere damit gewarnt haben. Bisher war dies nur über ein Reset der App möglich."
+                            "Seit der Version 1.13 haben Sie die Möglichkeit nach einer überstanden SARS-CoV-2 Infektion, die App wieder zu aktivieren und die Fortführung der Risiko-Ermittlung nach Teilung der eigenen Zufalls-IDs erneut zu starten. Dies bedeutet, dass Sie nun entscheiden können, ob und wann Sie die Risiko-Ermittlung wieder einschalten, nachdem Sie Ihre verschlüsselten Zufalls-IDs nach einem Positivtest geteilt und andere damit gewarnt haben. Bisher war dies nur über ein Reset der App möglich.",
+                            "Damit die App Ihnen die 'Infektionsrisiko'-Kachel wieder anzeigt, müssen Sie das <b>positive Testergebnis aus der App löschen</b>. Um dies zu tun tippen Sie bitte auf die Kachel, die das positve Testergebnis anzeigt und tippen dort auf 'Test löschen'. Anschließend wird das Testergebnis für 30 Tage in den Papierkorb verschoben und die 'Infektionsrisiko'-Kachel wird wieder angezeigt."

```

Comment:
"Um dies zu tun tippen Sie" (missing comma)
=> "Um dies zu tun, tippen Sie"

"positve" (typo)
=> "positive"

Reply:
Thanks, will be fixed in the next commit.

Reply:
Fixed with [`176ce94`](https://github.com/corona-warn-app/cwa-website/pull/2493/commits/176ce9417b360d8740c066d104f29073c5ce1bd8).